package.json

```diff
@@ -10,9 +10,13 @@
   "scripts": {
     "build": "web-ext build -s src/ && npm run build:clean",
     "build:clean": "git checkout src/manifest.json src/js/background/background.js src/popup.html",
-    "build:dev": "npm run build:clean && npm run config:dev && npm run package:chrome && npm run build",
-    "build:stage": "npm run build:clean && npm run config:stage && npm run package:chrome && npm run build",
-    "build:prod": "npm run build:clean && npm run config:prod && npm run package:chrome && npm run build",
+    "build:dev": "npm run build:clean && npm run config:dev && npm run build",
+    "build:stage": "npm run build:clean && npm run config:stage && npm run build",
+    "build:prod": "npm run build:clean && npm run config:prod && npm run build",
```

Comment:
Context: Removed the `package:chrome` script as this build script will no longer work for Chrome (without the extra script)